src/applications/static-pages/social-share-links.js

```diff
@@ -2,10 +2,9 @@
 Open share links in a new modal window
 */
 
-export function openShareLink() {
-  const shareLinks = document.querySelectorAll('.va-js-share-link');
+export function openShareLink(shareLinks) {
   const hasNavigatorShare = navigator.share !== undefined;
-  const metaTitle = document.querySelector("meta[name='title']").content;
+  const metaTitle = document.querySelector('title').content;
```

Comment:
I quickly searched for the difference between `meta[name='title']` and `title`. It seems like `title` is preferred, but I really don't know. Either way, I didn't see a Drupal page that contained `meta[name='title']`.